pkgs/top-level/aliases.nix

```diff
@@ -1002,6 +1002,7 @@ mapAliases ({
 
   # postgresql plugins
   cstore_fdw = postgresqlPackages.cstore_fdw;
+  jsonb_deep_sum = postgresqlPackages.jsonb_deep_sum;
```

Comment:
This alias is not necessary, please remove it. Besides that, LGTM

Reply:
I'm wondering though, why is this not needed and is in place and up to date for other extensions? :thinking:

Reply:
These alias are there for historical reasons. In a previous implementation of the PostgreSQL service, the extensions were available in the top level and to use them you had to something like:
`extraPlugins = [ (pkgs.postgis.override { postgresql = pkgs.postgresql_13; }) ]`
https://discourse.nixos.org/t/heads-up-upcoming-changes-to-the-postgresql-infrastructure-and-nixos-module/297/5
with the current implementation we don't really need the extensions be available at the top level.

Reply:
Thanks!